README.md

```diff
@@ -805,6 +805,13 @@ params.ReleaseNotes = "Release notes"
 params.ReleaseNotesSyntax = "plain_text"
 params.TargetProps = "key1=val1;key2=val2,val3"
 
+// Artifact path mapping:
+// After distribution, the artifact "source-repo/a/123.zip" destination in the edge node will be "target-repo/a-123.zip".
+// The spec's pattern is a wildcard with capturing groups surrounded by parenthesis. 
+// Respectfully, the spec target's replacement groups surrounded by curly brackets.
```

Comment:
I suggest modifying the above description to be:

// Be default, artifacts that are distributed as part of a release bundle, have the same path in their destination server 
// (the edge node) as the path they had on the distributing Artifactory server.
// You have however the option for modifying the tagtet path on edge node. You do this by defining the Target property as shown below.
// The Pattern property is a wildcard based pattern. Any wildcards enclosed in parenthesis in the pattern (source) 
// path can be matched with a corresponding placeholder in the target path, to determine the path and name 
// of the artifact, once distributed to the edge node.
// In the following example, the path in the edge node is similar the path in the source Artifactory server, except for the additional "dir" level at the root of the repository.
// Pattern: my-repo/(*)/a.zip
// Target: my-repo/dir/{1}/a.zip